johnwayodi/sm-api
# #161360318 Add jwt authorization to api endpoints

#### What does this PR do?
Adds jwt authorization to the api.
#### Description of Task to be completed?
1. Ensure tests are not broken by introduction of jwt auth
2. Host api on Heroku
3. Add more endpoints to api
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/sm-api.git`
2. Ensure you are in the cloned repo, *sm-api* by default. 
3. `git checkout ft-endpoints-authorization-161360318`
4. Run the application.
#### What are the relevant pivotal tracker stories?
#161360318
#### Questions:
What is the proper configuration for hosting on heroku?
